Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -349,6 +349,16 @@ extension FirefoxHomeViewController {
                 let window = UIApplication.shared.keyWindow
                 let safeAreaInsets = window?.safeAreaInsets.left ?? 0
                 insets += FirefoxHomeUX.MinimumInsets + safeAreaInsets
+                
+                // Ecosia
+                if UIDevice.current.userInterfaceIdiom == .phone {
```

Comment:
I think you can simplify this logic to:
`if UIDevice.current.userInterfaceIdiom == .pad || UIApplication.shared.statusBarOrientation.isLandscape {
  insets += frameWidth / 4
}`

So it does add insets when it's phone and portrait, right?

Reply:
Good idea